$NetBSD: patch-hackage-security.cabal,v 1.1 2021/04/24 13:32:22 pho Exp $

Fix build with GHC 9.
Fix build with base16-bytestring-1.0.*
Fix build with base64-bytestring-1.2.*

--- hackage-security.cabal.orig	2021-04-24 13:28:50.802251247 +0000
+++ hackage-security.cabal
@@ -110,13 +110,13 @@ library
                        Hackage.Security.Util.TypedEmbedded
                        Prelude
   -- We support ghc 7.4 (bundled with Cabal 1.14) and up
-  build-depends:       base              >= 4.5     && < 4.15,
-                       base16-bytestring >= 0.1.1   && < 0.2,
-                       base64-bytestring >= 1.0     && < 1.1,
+  build-depends:       base              >= 4.5     && < 5,
+                       base16-bytestring >= 0.1.1   && < 1.1,
+                       base64-bytestring >= 1.0     && < 1.3,
                        bytestring        >= 0.9     && < 0.11,
                        Cabal             >= 1.14    && < 1.26
                                       || >= 2.0     && < 2.6
-                                      || >= 3.0     && < 3.4,
+                                      || >= 3.0     && < 3.6,
                        containers        >= 0.4     && < 0.7,
                        ed25519           >= 0.0     && < 0.1,
                        filepath          >= 1.2     && < 1.5,
@@ -126,7 +126,7 @@ library
                        -- 0.4.2 introduces TarIndex, 0.4.4 introduces more
                        -- functionality, 0.5.0 changes type of serialise
                        tar               >= 0.5     && < 0.6,
-                       template-haskell  >= 2.7     && < 2.17,
+                       template-haskell  >= 2.7     && < 2.18,
                        time              >= 1.2     && < 1.10,
                        transformers      >= 0.3     && < 0.6,
                        zlib              >= 0.5     && < 0.7,
